eks-worker-al2.json

```diff
@@ -8,11 +8,13 @@
     "arch": null,
     "associate_public_ip_address": "",
     "aws_access_key_id": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_cli_version": "2.4.10",
     "aws_region": "us-west-2",
     "aws_secret_access_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
     "aws_session_token": "{{env `AWS_SESSION_TOKEN`}}",
     "binary_bucket_name": "amazon-eks",
     "binary_bucket_region": "us-west-2",
+    "cache_container_images": "true",
```

Comment:
I'm ready to ship this, thanks for all your work on this @bwagner5!

My only question is: should we default this to `false` to begin with? We can enable it for our official builds, but custom AMI users may want this to be opt-in, at least initially. @mmerkes you got an opinion here?

Reply:
I'd be fine with it either way. 

On one side, I'd expect the official images to use mostly defaults for reproducible builds from users. But I can also understand wanting to be cautious with the permissions updates and the additional storage overhead.

Reply:
Ideally, we should set this to false and allow feedback to come in. However, if we think it unlikely to actually get any feedback without enabling by default, the discussion is really just whether or not we're comfortable shipping this.

Reply:
I'll update it to default to false. I think the benefits will prompt users to enable this and we can gather feedback.